.github/workflows/pipelines-root.yml

```diff
@@ -31,7 +31,7 @@ on:
 
 env:
   PIPELINES_CLI_VERSION: v0.10.1
-  PIPELINES_ACTIONS_VERSION: v1.3.3
+  PIPELINES_ACTIONS_VERSION: mutexspike
```

Comment:
TODO - Update once actions PR (https://github.com/gruntwork-io/pipelines-actions/pull/24) is merged